src/test/java/uk/gov/hmcts/reform/ccd/client/mock/CcdWireMock.java

```diff
@@ -12,7 +13,7 @@
 import static org.springframework.http.HttpHeaders.AUTHORIZATION;
 
 public class CcdWireMock {
-    private static final WireMockServer SERVER = new WireMockServer(8081);
+    private static final WireMockServer SERVER = new WireMockServer(Options.DYNAMIC_PORT);
```

Comment:
Will this ever try to start up on a port already in use and error out? (not sure how this works if it keeps trying to start on a free port if theyre blocked or if it just fails on first attempt so i may be wrong!)

Reply:
I changed this to start on a random port because of that exact same reason - 8081 was being used by McAfee on my macbook and I kept getting port clashes

Documentation from 
https://wiremock.org/docs/getting-started/

says it picks random ports that are already free

```
Random port numbers
You can have WireMock (or more accurately the JVM) pick random, free HTTP and HTTPS ports (which is a great idea if you want to run your tests concurrently)
```